Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -179,6 +185,7 @@ public struct Configuration: Equatable {
             }
             self.init(dict: dict)!
             configurationPath = fullPath
+            hash = dict.bridge().hashValue
```

Comment:
Nah, apparently `NSDictionary` hash just returns the count. I've tested `dict.description.hashValue` locally and that seems to work surprisingly well.